src/bootstrap/src/ferrocene/dist.rs

```diff
@@ -9,11 +17,8 @@ use crate::ferrocene::doc::ensure_all_xml_doctrees;
 use crate::ferrocene::test_outcomes::TestOutcomesDir;
 use crate::t;
 use crate::utils::tarball::{GeneratedTarball, Tarball};
-use std::collections::BTreeMap;
-use std::fs;
-use std::path::{Path, PathBuf};
-use std::process::Command;
-use std::rc::Rc;
+
+pub(crate) use self::flip_link::FlipLink;
```

Comment:
Nit: rather than re-exporting I'd make the module `pub(crate)`. That way it's less maintenance when we add a new item to modules (we just have to add it to the list of steps in `core/builder.rs` rather than also `ferrocene/parent.rs`).

Reply:
Done